frontend/src/components/data-table/columns.tsx

```diff
@@ -307,8 +355,11 @@ function getFilterTypeForFieldType(
 function getCellStyleClass(
   justify: "left" | "center" | "right" | undefined,
   wrapped: boolean | undefined,
+  isSelected: boolean,
 ): string {
   return cn(
+    "cursor-pointer", // TODO: only when select cell?
+    isSelected && "bg-blue-200",
```

Comment:
can you use `bg-[var(--blue-2)]` or some number from there. this will be dark-mode compatible